src/components/navBar/navBar.scss

```diff
@@ -38,6 +38,11 @@ $md-nav-bar-height: 48px;
   &:hover {
     background-color: inherit;
   }
+
+  &[disabled] {
+    cursor: not-allowed;
+    opacity: 0.5;
```

Comment:
Changing `opacity` is bad here. It will cause a device to be rendered in 3D/Vector by most browsers instead the general 2D rendering.

You should apply the styling change to `navBar-theme.scss` instead, and perhaps use `foreground-3` for disabled items (I don't exactly remember what the right color should be).